Batteries/Data/String/Lemmas.lean

```diff
@@ -58,7 +58,8 @@ instance : Batteries.BEqOrd String := .compareOfLessAndEq String.lt_irrefl
 
 @[simp] theorem data_push (s : String) (c : Char) : (s.push c).1 = s.1 ++ [c] := rfl
 
-@[simp] theorem data_append (s t : String) : (s ++ t).1 = s.1 ++ t.1 := rfl
```

Comment:
Oh my god, I completely screwed that up. I meant to try to remove `cons_append` (since I oftentimes end up having lemmas that aren't necessary later on), and after seeing the tests pass, I just assumed that was all right, even though it was completely the wrong lemma... Big failure to read on my part, and just reverted that change